traffic_ops/traffic_ops_golang/cfgdiffs.go

```diff
@@ -0,0 +1,303 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/auth"
+
+	"github.com/jmoiron/sqlx"
+)
+
+const CfgDiffsPrivLevel = auth.PrivLevelReadOnly
+const CfgDiffsWritePrivLevel = auth.PrivLevelOperations
+
+type CfgFileDiffs struct {
+	FileName         string   `json:"fileName"`
+	DBLinesMissing   []string `json:"dbLinesMissing"`
+	DiskLinesMissing []string `json:"diskLinesMissing"`
+	ReportTimestamp  string   `json:"timestamp"`
+}
+
+type CfgFileDiffsResponse struct {
+	Response []CfgFileDiffs `json:"response"`
+}
+
```

Comment:
is this Response struct used in more than one place?  If not, should probably just define it where it's used rather than making it exported.